README_ru-RU.md

```diff
@@ -11,7 +11,7 @@
 
 **[Возможности](https://cpeditor.org/ru) · [Документация](https://cpeditor.org/ru/docs/) · [Список изменений](CHANGELOG.md) · [Сотрудничество](CONTRIBUTING_ru-RU.md) · [FAQ](https://cpeditor.org/ru/docs/faq/)**
 
-[English](README.md) · [中文](README_zh-CN.md)
+[English](README.md) · [简体中文](README_zh-CN.md) · [正體中文](README_zh-TW.md)
```

Comment:
It's better to keep the names the same around different version of README. Maybe change to `正體中文（臺灣）` which is used in English README?

Reply:
I prefer removing "（臺灣）" everywhere, or use "臺灣正體".

Reply:
In my opinion. as the traditional chinese do have two branches (HK and TW) (Maybe three, including MO), it's ok to keep the region. But maybe `繁體` instead of `正體`?

Reply:
Or maybe just ignore TW and use `繁體中文`? I don't think the difference between zh_HK and zh_TW matters much.

Reply:
What about "大陆简体" and "臺灣正體" like Wikipedia(`zh.wikipedia`, though)? Do we need "中文"?

Reply:
正體中文 is one of the preferred description of Chinese Traditional used in Taiwan FLOSS community. The main purpose of adding 臺灣 is to inform users the target audience of translation work. If we just want to divide into Simplified and Traditional, maybe we should use `zh-Hans` and `zh-Hant`?

Reply:
Actually I think it's not necessary to add plural types of Hant, so maybe remove the region and treat it as the only version of Hant?

Reply:
Different regions use different words and characters, so not providing a zh_HK version doesn't make the zh_TW version region-neutral.

I just think "正體中文（臺灣）" is a bit too long, we can use "正體中文" until we add zh_HK in the future, or use the shorter one "臺灣正體".

Reply:
Alright, 正體中文 seems more reasonable. I will take @ouuan 's advice.